### DISCUSSION TITLE
v2 React `serve` hanging up on ReactPropTypesSecret?

### DISCUSSION THREAD

**Discussion by AlexMoutonNoble** [parcel-bundler/parcel#6771] [Q&A]
Hi Folks
Porting a project to v2 parcel, i have build succeeding and now trying the dev environment.
My index loads and pulls down some number of css/js etc before printing this error to the console and dying.  ReactPropTypesSecret exports a single string that is almost comical...something about `SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED` 
esmodule-helpers seems to be checking the export for an 'init' key and then dereferencing by it and so I am fully boggled. how this error could happen since the exports should not have the key?

Are you somehow cursed with this knowledge? Is this something for parcel or should I go further down the stack here?

Thanks
Alex

```
Uncaught TypeError: Cannot read property 'init' of undefined
    at Object.init (ReactPropTypesSecret.js:12)
    at esmodule-helpers.js:16
    at Array.forEach (<anonymous>)
    at Object.exports.exportAll (esmodule-helpers.js:10)
    at Object.1gx6a.@sentry/browser (index.ts:1)
    at newRequire (index.6701a6e1.js:71)
    at localRequire (index.6701a6e1.js:83)
    at Object.dLPEP.react (index.jsx:2)
    at newRequire (index.6701a6e1.js:71)
    at index.6701a6e1.js:120
(anonymous) @ ReactPropTypesSecret.js:12
(anonymous) @ esmodule-helpers.js:16
exports.exportAll @ esmodule-helpers.js:10
1gx6a.@sentry/browser @ index.ts:1
newRequire @ index.6701a6e1.js:71
localRequire @ index.6701a6e1.js:83
dLPEP.react @ index.jsx:2
newRequire @ index.6701a6e1.js:71
(anonymous) @ index.6701a6e1.js:120
(anonymous) @ index.6701a6e1.js:143
```

**Comment by AlexMoutonNoble** [comment]
oh..i suppose i can pursue the @sentry/browser import?

**Comment by AlexMoutonNoble** [comment]
Removing @sentry/react avoids it but I wonder what I can tell them to have it be fixed?

**Reply by mischnic** [reply]
Please provide a code sample that reproduces the issue.

A similar issue: https://github.com/parcel-bundler/parcel/issues/6642
